akka-sample-cqrs-scala/src/main/scala/sample/cqrs/CqrsApp.scala

```diff
@@ -100,44 +110,31 @@ object CqrsApp {
 
   // FIXME very temporary test
   def testIt(system: ActorSystem): Unit = {
-    val entity1 = system.actorOf(TestEntity.props(), "entity1")
-    entity1 ! "a1"
-    entity1 ! "a2"
-    entity1 ! "a3"
-
-    val entity2 = system.actorOf(TestEntity.props(), "entity2")
-    entity2 ! "b1"
-    entity2 ! "b2"
 
-    val processor1 = system.actorOf(EventProcessor.props("tag1"), "processorTag1")
+    ShardedSwitchEntity(system).start()
```

Comment:
This must be done on all nodes at system startup time, since it's initializing sharding.